operator/option/config.go

```diff
@@ -256,6 +256,11 @@ const (
 	// IngressLBAnnotationPrefixes are the annotations which are needed to propagate
 	// from Ingress to the Load Balancer
 	IngressLBAnnotationPrefixes = "ingress-lb-annotation-prefixes"
+
+	// Disables operation of kubernet-related services/controllers.
+	// Intended for operating cilium with CNI-compatible orquestrators
+	// othern than Kubernetes.
+	DisableK8sServices = "disable-k8s-services"
```

Comment:
One nit, I would suggest using `enable-k8s-services` here and reversing the logic. Otherwise we regularly end up with double-negatives in the code like `!disableFoo` which contributors often find confusing.

Reply:
Sure, I'll update it to enable-k8-services, with default=true.